src/js/lib.js

```diff
@@ -373,7 +359,7 @@ vjs.TEST_VID = vjs.createEl('video');
  * @constant
  * @private
  */
-vjs.USER_AGENT = navigator.userAgent;
+var USER_AGENT = navigator.userAgent;
```

Comment:
It's fine for leaving it as const for now. switching to eslint can be another PR.